### ISSUE TITLE
Control has no label / is not focusable [curator-facing]  (severity 10)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1608]
Two issues associated with one component - the repeating "Contact emails" field on the Configuration - General form.

### The Contact emails input(s) have no label
`<label class="col-form-label col-md-2" for="exhibit_contact_emails">Contact emails</label>`
a) the "for" ID does not match the ID of the actual input(s).
b) each input needs a unique label

Possible solution:  
- [x] make the "for" id of the label match the first text input
- [x] give subsequent text inputs a unique `aria-label` (e.g. `aria-label="Contact email 2"`)

### The "Add new contact" button is not a focusable element 
`<span id="another-email" class="btn btn-sm btn-info">Add new contact</span>`

A `<span>` doesn't automatically get focus and doesn't respond to keyboard input (space or enter) even if it is added to the tab order. This should be a button.
